### DISCUSSION TITLE
can not search endpoint for graphql

### DISCUSSION THREAD

**Discussion by TYzzt** [apache/skywalking#8970] [Q&A] [answered]
when search endpoint all,we can get trace,but when search by graphql name get nothing

![image](https://user-images.githubusercontent.com/14325581/165722535-c41b2aec-8d53-453b-802e-db635dea460f.png)

![image](https://user-images.githubusercontent.com/14325581/165722990-5c1bfb6a-c3fb-498c-82a6-01b763cf4c36.png)

**Comment by wu-sheng** [comment, marked as answer]
This is an exit span operation name, not an endpoint.

**Reply by springmonster** [reply]
Hi @wu-sheng , I met the same problem after we integrated with GraphQL.
 
GraphQL uses '/graphql' as a unified endpoint and it becomes complex when we want to find out which method has a problem. We can't search endpoint anymore.

Do you have any suggestions about how can we find the GraphQL method just like we find endpoints in REST API?

Thanks very much!

**Reply by wu-sheng** [reply]
We just have the logic endpoint to metrics statistics, but for query trace, we don't have solution for now.

**Reply by springmonster** [reply]
Thanks for your reply

**Reply by TYzzt** [reply]
thanks
